packages/commons/src/file-manager/fileManager.ts

```diff
@@ -20,11 +20,6 @@ import {
 } from "./fileManagerErrors.js";
 
 export type FileManager = {
-  buildS3Key: (
-    path: string,
-    resourceId: string | undefined,
-    fileName: string
-  ) => string;
```

Comment:
Removing this since the `buildS3Key` function is not used outside the file manager, no need to export it